{
 "bad": [
  "AsyncLibraryNotFoundError",
  "asyncio.Task.current_task",
  "current_async_library",
  "current_async_library_cvar.reset",
  "current_async_library_cvar.set",
  "current_task",
  "sys",
  "sys.modules",
  "sys.version_info",
  "this_is_asyncio"
 ],
 "deps": {
  "AsyncLibraryNotFoundError": [],
  "asyncio": [
   "python/conda-forge/linux-64/python-3.7.0-h5001a0f_4",
   "python/conda-forge/linux-64/python-3.7.8-hffdb5ce_3_cpython",
   "python/conda-forge/linux-64/python-3.8.0-h357f687_1",
   "python/conda-forge/linux-64/python-3.8.5-h1103e12_4_cpython",
   "python/conda-forge/linux-64/python-3.8.5-h1103e12_5_cpython"
  ],
  "contextvars": [
   "contextvars/conda-forge/noarch/contextvars-2.1-py_0",
   "contextvars/conda-forge/noarch/contextvars-2.2-py_0",
   "contextvars/conda-forge/noarch/contextvars-2.4-py_0",
   "python/conda-forge/linux-64/python-3.7.8-hffdb5ce_3_cpython",
   "python/conda-forge/linux-64/python-3.8.5-h1103e12_4_cpython"
  ],
  "curio": [
   "curio/conda-forge/linux-64/curio-0.8-py35_0",
   "curio/conda-forge/noarch/curio-0.9-py_1002",
   "curio/conda-forge/noarch/curio-1.1-py_0",
   "curio/conda-forge/noarch/curio-1.2-py_0"
  ],
  "current_async_library": [],
  "current_async_library_cvar": [],
  "current_task": [],
  "pytest": [
   "pytest/conda-forge/linux-64/pytest-3.1.1-py27_0",
   "pytest/conda-forge/linux-64/pytest-3.10.0-py37_1000",
   "pytest/conda-forge/linux-64/pytest-3.2.2-py36_1",
   "pytest/conda-forge/linux-64/pytest-3.7.2-py35_0",
   "pytest/conda-forge/linux-64/pytest-4.0.2-py38_1001",
   "pytest/conda-forge/linux-64/pytest-4.6.1-py27_0",
   "pytest/conda-forge/linux-64/pytest-4.6.2-py36_0",
   "pytest/conda-forge/linux-64/pytest-5.1.2-py37_0"
  ],
  "sys": [],
  "this_is_asyncio": []
 }
}